src/app/header/header.component.spec.ts

```diff
@@ -25,7 +26,7 @@ describe('HeaderComponent', () => {
         NgxsModule.forRoot([]),
         HttpClientModule,
         RouterTestingModule,
-        MatProgressSpinnerModule,
+        MatProgressBarModule,
```

Comment:
I would rather added it to shared module since we are going to use it across the whole app